### PR TITLE
Added Fat Sting, made Fat and Unfat Self-Stingable.

### PR DESCRIPTION
**Pull request by N3X15**
# Executive Summary
This PR adds the Fat Sting, and allows both it and the unfat sting able to be used on yourself.

# Justification
Changelings have often found themselves in the position of assuming the appearance of someone who is overweight.  Until now, the only option they had to gain that weight was to run down to the kitchen to eat everything in sight.  In addition, switching from a fat appearance to a thin one no longer requires the assistance of another changeling.

# Testing
Tested by becoming a changeling and then using fat sting on myself.

# Changelog
:cl:
* rscadd: Changelings can use fat stings on themselves and others.
* rscadd: Changelings can now use unfat stings on themselves.

### CODE REVIEW

**Comment by Probe1** [comment]
This is the least thing wrong with changelings and there is a very long list of things wrong but holy fuck is stinging people and making them fat hilarious.

File: code/game/gamemodes/changeling/changeling_powers.dm [hunk 1]
```diff
@@ -943,7 +947,7 @@ var/list/datum/dna/hivemind_bank = list()
 		add_changeling_verb(verb_path)
 
 	to_chat(src, "<span class='notice'>We stealthily sting [T].</span>")
-	if(!T.mind || !T.mind.changeling)
+	if(!T.mind || !T.mind.changeling || allow_self)
```
Comment (PJB3005) [inline code/game/gamemodes/changeling/changeling_powers.dm]: Shouldn't this be `|| (allow_self && T == src)`?

**Review by PJB3005 — APPROVED** [review]

**Comment by carshalash** [comment]
Thanks fam.

**Comment by Dizor** [comment]
Well
Time to add lings to my antag pool.

**Comment by Probe1** [comment]
@N3X15 FLIX CONFLICK

**Pull request merged**